ballerina-tests/tests/01_listener_test.bal

```diff
@@ -26,3 +27,59 @@ function testInvalidMaxQueryDepth() returns error? {
     graphql:Error err = <graphql:Error>result;
     test:assertEquals(err.message(), "Max query depth value must be a positive integer");
 }
+
+graphql:Service greetingService = service object {
+    resource function get greeting() returns string {
+        return "Hello, World";
+    }
+};
```

Comment:
Let's move the services to the test_services file.

Reply:
Updated